serverAkkaHttp/src/main/scala/RPCRouter.scala

```diff
@@ -35,7 +35,7 @@ trait Router extends RPCServer with PathMacro with MetaDataMacro with LazyLoggin
 
   def exceptionHandler = ExceptionHandler {
     case e: FailException[_] =>
-      logger.error(e.getMessage, e)
+      logger.error(s"encoding error: ${e.getMessage}")
```

Comment:
add the stacktrace for debugging?

Reply:
The stacktrace looks like this: it's not super-informative.

<img width="1137" alt="41228175-ba668f30-6d77-11e8-89ab-b9ee9a01c11a" src="https://user-images.githubusercontent.com/2476080/41279888-ebbc7520-6e2d-11e8-8ec3-e0e57f689e7b.png">

Note that this error comes from autowire encoding (this is enforced by the type). The message is inherited from autowire.

I agree this is not a super-clean solution but:
1. The stacktrace is not useful (it's also a little annoying in bobafett)
2. I don't have better solutions at the moment

This is not solving the root issue (`null` 😡). We can spend some more time to understand that.
It doesn't seem to cause real issues though.

Reply:
👍